lib/ts/querier.ts

```diff
@@ -42,15 +45,25 @@ export default class Querier {
             preAPIHook,
             postAPIHook
         );
-        return await result.json();
+
+        let fetchResponse = result.clone();
+        let json = await result.json();
+
+        return {
+            json,
+            fetchResponse,
+        };
     };
 
     post = async <T>(
         path: string,
         config: RequestInit,
```

Comment:
You need to check if the config.body === undefined, then throw an error like in here: https://github.com/supertokens/supertokens-auth-react/commit/74f0218c84977338b371c125a5fa8cf5dbeadaea

Reply:
Done